Configuration/AlCa/python/autoCond.py

```diff
@@ -44,6 +44,8 @@
     # GlobalTag for Run2 HLT for HI (not 2018 HI): it points to the online GT
     'run2_hlt_hi'       :   '101X_dataRun2_HLTHI_frozen_v9',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2017 (and 0,0,~0-centred beamspot)
+    'run3_data'         :   '106X_dataRun3_Express_Candidate_2019_05_24_21_36_45',
+    # GlobalTag for Run3 data relvals
```

Comment:
The comments in L46 and L48 should be switched so that the comment is above the corresponding key-value pair. Please try to maintain the same whitespace in the unchanged lines to avoid irrelevant merge conflicts.

Reply:
Hi @christopheralanwest , thanks for your comments (sorry for the delay, CMS week..) We should consider having prompt reconstruction in MWGR, do you have a name of a GT I should use  ?  (I will also use un3_data_promptlike as you recommended)   - And for the line switching , thanks for spotting , it was not intentionnal to change , I will fix in a next commit. Thanks again .